ftosv/docker/launch.py

```diff
@@ -146,7 +146,7 @@ def bootstrap_config(self):
         self.logger.info("applying bootstrap configuration once system is ready")
         self.wait_write("", None)
 
-        self.wait_write("configure", wait="System is ready!!")
+        self.wait_write("configure", wait="OS10#")
```

Comment:
hi @ssasso 

Won't this make OS 9 and earlier systems non-bootable?

Reply:
But from what I can see this "flavour" is only for OS10 (Dell OS9 is a totally different NOS).

Every OS10 version default prompt is *OS10#*. But they removed the string "system is ready".

Reply:
ahhh
super, lgtm